turbinia/config/logger.py

```diff
@@ -38,6 +39,12 @@ def setup(need_file_handler=True, need_stream_handler=True):
   # Eliminate double logging from root logger
   logger.propagate = False
 
+  # Add turbinia logs to syslog
+  syslog_handler = logging.handlers.SysLogHandler(address='/dev/log')
+  syslog_handler.formatter = logging.Formatter(
+      'Turbinia: { "loggerName":"%(name)s", "levelNo":"%(levelno)s", "levelName":"%(levelname)s", "message":"%(message)s"}'
```

Comment:
Minor nit:  It looks like the line length is too long here.

Reply:
Yapf formatted it this way. lmk if you want me to change that

Reply:
Yeah, yapf doesn't know how to break apart strings, but you can just do it manually like:
```
logging.Formatter(
    'foo '
    'bar')
```

If you don't break it up you could also add a lint disable comment, but breaking it up is probably better.

Reply:
Done